wagtail/wagtailsearch/management/commands/update_index.py

```diff
@@ -35,7 +42,7 @@ def update_backend(self, backend_name, object_list):
         index = rebuilder.start()
 
         for model, queryset in object_list:
-            self.stdout.write(backend_name + ": Indexing model '%s.%s'" % (
+            self.stdout.write(backend_name + u": Indexing model '%s.%s'" % (
```

Comment:
Could use use ``from __future__ import unicode_literals`` at the top of this file instead?